service_capacity_modeling/models/org/netflix/counter.py

```diff
@@ -92,33 +92,52 @@ def extra_model_arguments_schema() -> Dict[str, Any]:
     def compose_with(
         user_desires: CapacityDesires, extra_model_arguments: Dict[str, Any]
     ) -> Tuple[Tuple[str, Callable[[CapacityDesires], CapacityDesires]], ...]:
-        stores = [("org.netflix.evcache", lambda x: x)]
-        if extra_model_arguments["counter.mode"] != NflxCounterMode.best_effort.value:
+        stores = []
 
+        if extra_model_arguments["counter.mode"] == NflxCounterMode.best_effort.value:
+            stores.append(("org.netflix.evcache", lambda x: x))
+        else:
+            # Shared evcache cluster is used for eventual and exact counters
             def _modify_cassandra_desires(
                 user_desires: CapacityDesires,
             ) -> CapacityDesires:
                 modified = user_desires.model_copy(deep=True)
+                counter_cardinality = extra_model_arguments["counter.cardinality"]
 
                 # counts per second
-                cps = user_desires.query_pattern.estimated_write_per_second
+                counter_deltas_per_second = (
+                    user_desires.query_pattern.estimated_write_per_second
+                )
+
+                # low cardinality : rollups happen once every 100 seconds
```

Comment:
Lets update based on `CounterConfig` in `cde-dgw-counter` repository. There should be a ection on how long we coalesce depending on the cardinality. Other related information that drives counter provisioning should be in there as well.